src/reader/TranslatableWord.js

```diff
@@ -88,14 +122,41 @@ export default function TranslatableWord({
   return (
     <>
       <z-tag>
-        {word.translation && (
+        {word.translation && isVisible && (
           <z-tran
             chosen={word.translation}
             translation0={word.translation}
             ref={refToTranslation}
-            onClick={(e) => toggleAlterMenu(e, word)}
           >
-            <span className="arrow">▼</span>
+            <div className="translationContainer">
+              <span onClick={(e) => toggleAlterMenu(e, word)}>
+                {word.translation}
+              </span>
+              {showingAlterMenu ? (
+                <span
+                  className="arrow"
+                  onClick={(e) => toggleAlterMenu(e, word)}
+                >
+                  ▲
```

Comment:
the only thing that's different between these two branches is the arrow. so factor out the span and put the ternary operator only for the arrow. or?

Reply:
That's true, I have changed it!